### ISSUE TITLE
JUnit report has wrong duration

### ISSUE THREAD

**Issue by ahilsend** [postmanlabs/newman#867]
1. Newman Version: v3.3.1 (from the docker image)
 2. OS details (type, version, and architecture): newman_alpine33 docker image: node 4.3.0 on Alpine 3.3
 3. CLI
 3. Did you encounter this recently, or has this bug always been there: n/a
 4. Expected behaviour: 
 5. Command / script used to run Newman:
 6. Sample collection, and auxilliary files (minus the sensitive details):

The output from the JUnit report has the wrong time unit for the duration field.
According to this [JUnit.xsd](http://windyroad.com.au/dl/Open%20Source/JUnit.xsd) the field 'time' is defined as:

```xml
<xs:attribute name="time" type="xs:decimal" use="required">
    <xs:annotation>
        <xs:documentation xml:lang="en">Time taken (in seconds) to execute the test</xs:documentation>
    </xs:annotation>
</xs:attribute>
```

The values in the JUnit report however are the milliseconds. Meaning if the request took 150ms, the time property will have the value 150.

**Comment by bteng22** [comment]
Took a stab at resolving this issue in PR #917. Seemed to be a small change in the JunitReporter when getting the execution responseTime. Any feedback is more than welcomed 👍

**Comment by kunagpal** [comment]
Released with Newman `v3.5.0`

**Comment by kaupadhy** [comment]
Hi, I have version 3.5.2 but still facing the issue of incorrect duration. It is showing only ms and not seconds and minutes in duration
